client/src/main/java/com/pk/App.java

```diff
@@ -1,36 +1,70 @@
-package com.pk;
+package com.pk.checkers;
 
 import javafx.application.Application;
-import javafx.fxml.FXMLLoader;
+import javafx.scene.Group;
 import javafx.scene.Parent;
 import javafx.scene.Scene;
+import javafx.scene.layout.Pane;
 import javafx.stage.Stage;
 
-import java.util.Locale;
-import java.util.ResourceBundle;
-
-/**
- * Hello world!
- *
- */
 public class App extends Application {
 
-  public Stage stage;
-  public ResourceBundle translations;
-  public Locale locale;
-
-  @Override
-  public void start(Stage primaryStage) throws Exception {
-    stage = primaryStage;
-    locale = new Locale("pl_PL");
-    translations = ResourceBundle.getBundle("translations", locale);
-    Parent root = FXMLLoader.load(getClass().getClassLoader().getResource("MainMenuView.fxml"), translations);
-    primaryStage.setTitle("Checkers");
-    primaryStage.setScene(new Scene(root, 800, 800));
-    primaryStage.show();
-  }
-
-  public static void main(String[] args) {
-    launch(args);
-  }
+    public static final int TILE_SIZE = 100;
+    public static final int WIDTH = 8;
+    public static final int HEIGHT = 8;
+
+    private Tile[][] board = new Tile[WIDTH][HEIGHT];
```

Comment:
Add Tile import

Reply:
To add to that best would be to change it to List